Add Support Us section with styled CTA linking to /support

si-rak opened the pull request:
### What does this PR do?

Adds the “Support Us” section with a heading, description text, and a Support button based on the Figma design. The button links to the `/support` page.

---

### Description of Task to be completed

- Created the Support Us section using the provided Figma design
- Used an existing `ReusableButton` component from a previous team
- Added custom styles just to this button to fix the hover issue (white text on white background)
- Made sure it looks good on all screen sizes and works with keyboard navigation
- The button links to `/support` using React Router’s `<Link>`

---

### How should this be manually tested?

1. Run `npm run dev`
2. Scroll to the Support Us section on the event page
3. Check layout, responsiveness, and hover color
4. Click the Support button and make sure it routes to `/support`

---

### Any background context?

- The `/support` page doesn’t exist yet, so clicking the button will show 404 until the SupportPage is built
- I didn’t change any global button styles — only added local overrides in this component
- Also restored files I had accidentally deleted before pushing

sanakhuram committed:
✨ Add EventVisuals component and integrate it into Event page

Niksubishi committed:
Rikke's EventHeader Updates

Niksubishi committed:
Small eslint fixes.

Niksubishi committed:
Update EventHeader.jsx

Niksubishi committed:
Merge branch 'feature/event-page' into feature/event-visuals

Niksubishi committed:
Merge pull request #1148 from NoroffFEU/feature/event-visuals

 Added EventVisuals Component with Lightbox & Insert into Event Page

Inamarie84 committed:
Add ReasonsToAttend component and integrate into Event page

si-rak committed:
Add Support Us section with styled CTA linking to /support

Niksubishi committed:
Moved file and removed placeholder

File moved to components/event
Removed WhyAttend placeholder

Niksubishi committed:
Create ReasonsToAttend.jsx

Niksubishi committed:
Merge pull request #1151 from NoroffFEU/feature/event-reasons

Added ReasonsToAttend component and integrate into Event page

Niksubishi committed:
Delete ReasonsToAttend.jsx

Niksubishi commented:
@si-rak The PR cannot be approved and merged at this point. There are too many conflicts with feature/event-page that need addressing. 

I suggest pulling the latest pushes to the feature/event-page branch, then adding your component. 
Things to do:

- [ ] Pull latest commits to feature/event-page.
- [ ] Add in your component (please add to the components/event folder).
- [ ] Speak with your fellow developers about importing your component into current event-page.
- [ ] Update me when these are completed.

Feel free to contact me or Linn to discuss this if there is any confusion.

si-rak commented:
Oh! didn’t realize there were that many conflicts. I’ll pull the latest
changes, resolve the conflicts and keep you posted once everything’s done.
Tomorrow may be.

fre. 30. mai 2025 kl. 18:33 skrev Nik Bishop ***@***.***>:

> *Niksubishi* left a comment (NoroffFEU/musikkforandrerliv.no#1164)
> <https://github.com/NoroffFEU/musikkforandrerliv.no/pull/1164#issuecomment-2922849501>
>
> @si-rak <https://github.com/si-rak> The PR cannot be approved and merged
> at this point. There are too many conflicts with feature/event-page that
> need addressing.
>
> I suggest pulling the latest pushes to the feature/event-page branch, then
> adding your component.
> Things to do:
>
>    - Pull latest commits to feature/event-page.
>    - Add in your component (please add to the components/event folder).
>    - Speak with your fellow developers about adding your component into
>    current event-page.
>    - Update me when these are completed.
>
> Feel free to contact me or Linn to discuss this if there is any confusion.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NoroffFEU/musikkforandrerliv.no/pull/1164#issuecomment-2922849501>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BEONBWNU5APTDPJ7INOOSID3BCB7NAVCNFSM6AAAAAB6EEDX3GVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSMRSHA2DSNJQGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

si-rak committed:
Resolved merge conflict in Event.jsx and added SupportUsSection

si-rak commented:
@Niksubishi 
The requested updates are completed:

- Resolved the merge conflict in Event.jsx
- Added SupportUsSection to src/component/event
- Updated the branch with feature/event-page

si-rak commented:
Replaced by a clean PR: #1182

This PR was closed due to a misunderstanding in component placement and file structure, as well as unresolved merge conflicts.  
A clean version of the Support Us section was implemented in the correct location, and pushed from `feature/event-support` into `feature/event-page`.

Thanks for your patience!
